lib/loader/index.d.ts

```diff
@@ -1,118 +1,121 @@
 /// <reference lib="esnext.bigint" />
 
 interface ResultObject {
-  module: WebAssembly.Module,
+  module: WebAssembly.Module
```

Comment:
Comma here is fine

Reply:
I know. But I decide remove any optional terminators in definition file due to it can't cause to problems

Reply:
It reduce verbosity & size

Reply:
Is code style chaos between files though. Not liking it :)

Reply:
Ok, I revert semicolons